### PR TITLE
Fix some multi_queues_test problems

### PR DESCRIPTION
**Pull request by PaulYuuu**
max_queues:
1. Add "vt_ulimit_nofile" to fix "Too many  open files"
2. Modify the method to get "top_info" to fix "top: pid limit (20) exceeded"

cpu_affinity:
1. Modifiy the "cmd" to fix to get an empty "irq_statics" when irq_number > 99
2. Restart "irqbalance.service" finally

others:
1. Sleep 2min to make the results more accurate
2. Use "top -i" to get the correctly running pid.

ID: 1367010 1647284

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@pezhang @xiywang Please help review, thanks.

File: generic/tests/cfg/multi_queues_test.cfg [hunk 1]
```diff
@@ -54,6 +54,7 @@
                     queues = ${smp_fixed}
                     vhostfds_len = ${smp_fixed}
                     tapfds_len = ${smp_fixed}
+                    vt_ulimit_nofile = 8192
```
Comment (xiywang) [inline generic/tests/cfg/multi_queues_test.cfg]: Is this enough? I would suggest 10240.

Comment (PaulYuuu) [reply generic/tests/cfg/multi_queues_test.cfg]: I tried，8192 is enough for `max_queues`. 
```bash
# cat /proc/sys/fs/file-nr
4132	0	12265804
# lsof -p `pgrep qemu-kvm` | wc -l
2145
```

**Comment by PaulYuuu** [comment]
Waiting for the developer confirmation, will update some steps for each scenario.

**Comment by PaulYuuu** [comment]
Latest results after applying this patch. (Note: 2 cases failed due to the product bug)
 (1/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.with_netperf.smp1: FAIL: Run 1 netperf session, but 2 queues works (255.96 s)
 (2/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.with_netperf.smp4.single_netperf_session: FAIL: Run 1 netperf session, but 3 queues works (233.78 s)
 (3/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.with_netperf.smp4.multiple_netperf_session: PASS (306.87 s)
 (4/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.with_netperf.max_queues: PASS (305.70 s)
 (5/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.with_netperf.cpu_affinity: PASS (317.86 s)
 (6/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.invalid_queues_number.upper_border: PASS (2.88 s)
 (7/7) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.ppc64le.io-github-autotest-qemu.multi_queues_test.invalid_queues_number.lower_border: PASS (2.51 s)

**Comment by PaulYuuu** [comment]
@pezhang Could you help to review this one? Thank you very much.

**Comment by pezhang** [comment]
LGTM.

Acked by: Pei Zhang pezhang@redhat.com

**Comment by vivianQizhu** [comment]
@pezhang Please provide test result on x86 host, thanks.

**Comment by pezhang** [comment]
Testing on x86_64 PASS.

 (1/5) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.smp1.q35: PASS (347.97 s)
 (2/5) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.smp4.single_netperf_session.q35: PASS (278.88 s)
 (3/5) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.smp4.multiple_netperf_session.q35: PASS (274.18 s)
 (4/5) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.max_queues.q35: PASS (283.85 s)
 (5/5) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.cpu_affinity.q35: PASS (287.08 s)

**Pull request merged**